src/main/kotlin/net/atos/client/sd/model/wizard/DepositDeclarations.kt

```diff
@@ -2,10 +2,20 @@
  * SPDX-FileCopyrightText: 2024 Lifely
  * SPDX-License-Identifier: EUPL-1.2+
  */
-package net.atos.client.sd.model
+package net.atos.client.sd.model.wizard
 
 import jakarta.json.bind.annotation.JsonbProperty
 
+open class Deposit {
```

Comment:
Why not a `data class` instead of an `open class`? We can then also still add `@AllOpen` if needed (for Weld/Wildfly).